### PR TITLE
README.md: added documentation about issues, logs and help

### PR DESCRIPTION
**Pull request by examon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,24 @@ for more infomation and documenation.
...
+  - Leapp actors: [https://github.com/oamg/leapp-repository/issues/new/choose](https://github.com/oamg/leapp-repository/issues/new/choose)
+
+- When filing an issue, include:
+  - How to reproduce it
+  - The logs `/tmp/leapp-report.txt`, `/tmp/download-debugdata` and `/var/log/upgrade.log`
+  - The `/var/lib/leapp/leapp.db` file
+
+### Where can I seek help?
+We’ll gladly answer your questions and lead you to through any troubles with the
+actor development. You can reach us, [the OS and Application Modernization Group](https://mojo.redhat.com/groups/os-app-modernization/)
```
Comment (jmikovic) [inline README.md]: I don't think it makes sense mojo link on github

File: README.md [hunk 2]
```diff
@@ -6,6 +6,24 @@ for more infomation and documenation.
...
+by these means:
```
Comment (jmikovic) [inline README.md]: I would put it to one sentence, You can reach us at IRC ...

Why have a list if there is only one thing in it

File: README.md [hunk 3]
```diff
@@ -6,6 +6,24 @@ for more infomation and documenation.
...
 
+## Troubleshooting
+
+### Where can I report an issue or RFE related to the framework or other actors?
+- GitHub issues are preferred:
+  - Leapp framework: [https://github.com/oamg/leapp/issues/new/choose](https://github.com/oamg/leapp/issues/new/choose)
+  - Leapp actors: [https://github.com/oamg/leapp-repository/issues/new/choose](https://github.com/oamg/leapp-repository/issues/new/choose)
+
+- When filing an issue, include:
+  - How to reproduce it
```
Comment (jmikovic) [inline README.md]:
```suggestion
  - Steps to reproduce the issue
```

File: README.md (hunk 1 above)
Comment (vinzenz) [reply README.md]: indeed

**Pull request merged**